### PR TITLE
Additional comments for EPS on reason[x] elements

### PR DESCRIPTION
**Pull request by omarisgreat**
Additional changes to extension slicing have been automatically added by Forge

### CODE REVIEW

**Comment by omarisgreat** [comment]
Comments to only be added against old NHSD IG. Reviewed by Rob Gooch from DigiMeds perspective

File: StructureDefinition/NHSDigital-MedicationRequest.StructureDefinition.xml [hunk 1]
```diff
@@ -86,31 +86,18 @@
...
-    <element id="MedicationRequest.extension">
-      <path value="MedicationRequest.extension" />
-      <slicing>
-        <discriminator>
-          <type value="value" />
-          <path value="url" />
-        </discriminator>
-        <rules value="open" />
-      </slicing>
-      <min value="1" />
```
Comment (digitalhealthmartin) [inline StructureDefinition/NHSDigital-MedicationRequest.StructureDefinition.xml:98]: You mentioned that Forge removed slicing info. However, you sure you want to remove min cardinality of 1?

File: StructureDefinition/NHSDigital-MedicationRequest.StructureDefinition.xml [hunk 2]
```diff
@@ -135,13 +121,16 @@
...
-      <min value="0" />
       <type>
         <code value="Extension" />
         <profile value="https://fhir.nhs.uk/StructureDefinition/Extension-DM-PrescriptionStatusHistory" />
       </type>
       <mustSupport value="true" />
     </element>
+    <element id="MedicationRequest.extension:prescriptionTaskStatusReason.extension">
+      <path value="MedicationRequest.extension.extension" />
+      <min value="1" />
```
Comment (digitalhealthmartin) [inline StructureDefinition/NHSDigital-MedicationRequest.StructureDefinition.xml:132]: Is this right to add min cardinality of 1?

File: StructureDefinition/NHSDigital-MedicationRequest.StructureDefinition.xml (hunk 1 above)
Comment (omarisgreat) [reply StructureDefinition/NHSDigital-MedicationRequest.StructureDefinition.xml:98]: It was automatically recalculated within Forge, none of the individual slices on extension had a cardinality of 1 so there is no reason for the extension element to have a cardinality of 1 (unless you want to specify any of the below, but I can't see that being the case for this resource).

File: StructureDefinition/NHSDigital-MedicationRequest.StructureDefinition.xml (hunk 2 above)
Comment (omarisgreat) [reply StructureDefinition/NHSDigital-MedicationRequest.StructureDefinition.xml:132]: This is required as the extension is a complex extension and doesn't have a simple 'value', again recalculated by Forge and seems correct.

**Review by digitalhealthmartin — APPROVED** [review]

**Pull request merged**